cmd/generate/runner.go

```diff
@@ -44,6 +49,8 @@ func (r *runner) Run(cmd *cobra.Command, args []string) error {
 func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) error {
 	var err error
 
+	r.flag.Name = addNameSuffix(r.flag.Name)
```

Comment:
Won't it also add the suffix to the App CR name? I think that one should stay untouched. WDYT?